examples/wakaama/lwm2mconfig.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2018 Beduino Master Projekt - University of Bremen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Configurable parameters for Wakaama example client.
+ *
+ * @author      Christian Manal <manal@uni-bremen.de>
+ *
+ * @}
+ */
+
+
+#ifndef LWM2MCONFIG_H_
+#define LWM2MCONFIG_H_
+
+/*
+ * RIOT debugging
+ */
+#define ENABLE_DEBUG (1)
+#include "debug.h"
+
+/*
+ * DTLS debug log level. One of the following:
+ *    DTLS_LOG_EMERG
+ *    DTLS_LOG_ALERT
+ *    DTLS_LOG_CRIT
+ *    DTLS_LOG_WARN
+ *    DTLS_LOG_NOTICE
+ *    DTLS_LOG_INFO
+ *    DTLS_LOG_DEBUG
+ */
+#ifndef DTLS_LOG_LEVEL
+#define DTLS_LOG_LEVEL DTLS_LOG_CRIT
+#endif
+
+/*
+ * The name the device registers at the LwM2M server.
+ */
+#define LWM2M_DEVICE_NAME "testRIOTDevice"
+
+/*
+ * Device object lifetime on the server.
+ */
+#define LWM2M_DEVICE_TTL 300
+
+/*
+ * LwM2M server URI to register/bootstrap with.
+ * The host part of the URI MUST be a valid IPv6 address. Host names
+ * can not be resolved at this time.
+ */
+#define LWM2M_SERVER_URI "coaps://[fe80::5838:f4ff:fed6:6805]"
```

Comment:
Probably this should be inside of  #ifndef clause. Thus, we can define the server at fly.

Reply:
The idea was to edit the header before building (as I've described in the Readme file). If you think it's better to have these things in the Makefile, I might as well get rid of lwm2mconfig.h.

Reply:
My comment with this specific issue is not absolute but a reference.  IMHO this approach make the things more easy for testing. 

lwm2mconfig.h  could have default values. By example it could be `coaps://[fd00:dead:beef::1]` due that is the default value when `dist/tools/ethos/start_network.sh` is (called)[https://github.com/RIOT-OS/RIOT/blob/master/dist/tools/ethos/start_network.sh#L11].